src/Workspaces/Core/Portable/Rename/RenameLocation.ReferenceProcessing.cs

```diff
@@ -360,6 +360,13 @@ void AddRenameLocationIfNotGenerated(Location location, bool isRenamableAccessor
 
             internal static async Task<IEnumerable<RenameLocation>> GetRenamableReferenceLocationsAsync(ISymbol referencedSymbol, ISymbol originalSymbol, ReferenceLocation location, Solution solution, CancellationToken cancellationToken)
             {
+                // We won't try to update references in source generated files; we'll assume the generator will rerun
+                // and produce an updated document with the new name.
+                if (location.Document is SourceGeneratedDocument)
```

Comment:
We _could_ update the underlying FAR engine to avoid even doing the search, but that's a fairly invasive change, and writing a custom collector from the streaming results would also work but is a bunch of extra work. Unless we're seeing performance issues, best to keep the fix simple.